cmd/disk-cache.go

```diff
@@ -641,7 +640,7 @@ func newServerCacheObjects(ctx context.Context, config cache.Config) (CacheObjec
 	c := &cacheObjects{
 		cache:       cache,
 		exclude:     config.Exclude,
-		triggerHits: config.TriggerHits,
+		triggerHits: config.After,
 		migrating:   migrateSw,
```

Comment:
Isn't it confusing to have one variable .After and use it as TriggerHits, can we just keep it as Hits?